src/css/about.css

```diff
@@ -148,6 +152,14 @@
         height: 243px;
         border-radius: 30px;
     }
+
+    .about-picture-mobile{
```

Comment:
this styling is wrong, there is only one picture, use media queries with one class (there is only one image!) for proper styling for different screen sizes (please look again how it is done in prop-leave-app-section)

Reply:
what about ::after pseudo element display, it is also not corrected